iOS/MVC/README.md

```diff
@@ -0,0 +1,46 @@
+# MVC
+
+## Model ViewController
+
+### What is MVC?
+
+MVC is a software development pattern made up of three main objects:
+#### model
+The Model is where your data resides. Things like persistence, model objects, parsers, managers, and networking code live there.
+#### View
+The View layer is the face of your app. Its classes are often reusable as they don’t contain any domain-specific logic. For example, a UILabel is a view that presents text on the screen, and it’s reusable and extensible.
+#### Controller
+The Controller mediates between the view and the model via the delegation pattern. In an ideal scenario, the controller entity won’t know the concrete view it’s dealing with. Instead, it will communicate with an abstraction via a protocol. A classic example is the way a UITableView communicates with its data source via the UITableViewDataSource protocol.
+
+### Implementing the MVC
+So First of all you have to know that if the MVC Design pattern implements correctly it can help you code and debug faster . In MVC model shouldn't know about the Views and also the Views shouldn't know about the model this means for example : View expects a model to show (that's it!) and also for the Model it doesn't matter that the View design or how it's gonna show and etc. here are some things that you should consider and follow to have a maintainable code: 
+1- the controller should not manage every things :
```

Comment:
This line should be promoted to a topic.

It should be `#### 1. The controller should not manage everything.`

Reply:
#### added to the numbers